tests/unit/components/sl-grid-test.js

```diff
@@ -334,6 +334,68 @@ test( 'Window resize triggers updateHeight() with "auto" width', function( asser
     );
 });
 
+test( 'Grid adds and removes events from the correct namespace', function( assert ) {
+    const component = this.subject({ columns, content, height: 'auto' });
+    const contentElement = this.$( '.list-pane .content' );
+    const windowElement = Ember.$( window );
+    const jQueryData = Ember.get( Ember.$, '_data' );
```

Comment:
Same comment left on similar logic in sl-date-picker-test.js file applies here as well.